app/controllers/openid_connect/discovery_controller.rb

```diff
@@ -0,0 +1,33 @@
+module OpenidConnect
+  class DiscoveryController < ApplicationController
+    def webfinger
+      jrd = {
+        links: [{
+                  rel: OpenIDConnect::Discovery::Provider::Issuer::REL_VALUE,
+                  href: File.join(root_url, "openid_connect")
+                }]
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.